Ports/cowsay/package.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='cowsay'
+version='3.04'
+depends=('perl5')
+useconfigure='false'
+files="https://github.com/tnalpgge/rank-amateur-cowsay/archive/refs/tags/cowsay-${version}.tar.gz cowsay-${version}.tar.gz d8b871332cfc1f0b6c16832ecca413ca0ac14d58626491a6733829e3d655878b"
+workdir="rank-amateur-cowsay-cowsay-${version}/"
+
+prefix='/usr/local'
+bin_path="${prefix}/bin"
+cow_path_prefix="${prefix}/share"
+perl_executable="${bin_path}/perl"
+
+build() {
+    run sed -i "s#%PREFIX%#${prefix}#" cowsay
+    run sed -i "s#%BANGPERL%#!${perl_executable}#" cowsay
```

Comment:
```suggestion
    run_replace_in_file "s#%PREFIX%#${prefix}#" cowsay
    run_replace_in_file "s#%BANGPERL%#!${perl_executable}#" cowsay
```
I have to either add gnu-sed to my path on macOS, or use the suggested change, else I get an error.